sensorhub-driver-android/src/main/java/org/sensorhub/impl/sensor/android/AndroidSensorsDriver.java

```diff
@@ -51,63 +56,66 @@
 import android.os.Build;
 import android.provider.Settings.Secure;
 
+//import androidx.camera
+import android.hardware.camera2.CameraManager;
```

Comment:
Can you remove these imports. I think they are unused

Reply:
pushed commit to restore previous version of the file and one to optimize the commits